css/main.css

```diff
@@ -40,13 +40,13 @@ body
 {
 	position:absolute;
 	left:10000px;
-	bottom:135px;
+	bottom:500px;
```

Comment:
@OrionStar25 That's cuz you are defining it in pixels, see how the original car's bottom property has been specified. 
```css
#car
 {
 	position:fixed;
 	left:20%;
 	bottom:17%;
 	height:6vw;
 	width:14vw;
 	background:url('../images/car.png');
 	background-size: contain;
 	background-repeat: no-repeat;
 	z-index: 100;
 
 }
```